FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKInternalUtility.m

```diff
@@ -114,7 +114,9 @@ + (unsigned long)currentTimeInMilliseconds
 {
   struct timeval time;
   gettimeofday(&time, NULL);
-  return (time.tv_sec * 1000) + (time.tv_usec / 1000);
+    long second = (time.tv_usec / 1000);
+    long long timeL = time.tv_sec;
+    return (timeL * 1000) + (time.tv_usec / 1000);
```

Comment:
This is still going to return an unsigned long, how would it prevent an overflow?